FrontEnd/main/src/components/cookieacception/CookieMod.js

```diff
@@ -0,0 +1,54 @@
+import { Cookies } from 'react-cookie';
```

Comment:
It's preferable to use a `.jsx` extension for react components
https://stackoverflow.com/a/76491107/9918279

Reply:
I'll remember it. Thanl you. Done